packages/ember-auto-import/ts/auto-import.ts

```diff
@@ -75,6 +79,20 @@ export default class AutoImport implements AutoImportSharedAPI {
     this.consoleWrite = (...args) => addonInstance.project.ui.write(...args);
   }
 
+  installAppFilter(_host: AppInstance) {
+    // TODO upstream this type change to @embroider/shared-internals
+    let host: AppInstance & {
+      trees: {
+        app: Node;
+      };
+    } = _host as any;
```

Comment:
Since you force the type, could this work/be considered:

```suggestion
    let host = _host as AppInstance & {
      trees: {
        app: Node;
      };
    };
```